department-of-veterans-affairs/va.gov-cms
# VACMS-10793: new table audit

## Description
Closes #10793.

## Testing done
Tested locally. 

## Screenshots


## QA steps

Navigate to Admin Content. Click on Content Audit tool. You should see a new tab called "Tables(New)". Click on it. You will see a report of all the tables in the CMS which contains the Table ID, Published state, The table body, VA.gov url, Node link, Content type and Section. You should also be able to click on and download a csv version of the report. 

### Definition of Done

- [ ] Report is visible and functioning.
- [ ] CSV report downloads.

### Select Team for PR review



### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
